ee/ucp/kubernetes/layer-7-routing.md

```diff
@@ -181,17 +277,17 @@ spec:
         name: sysctl
         securityContext:
           privileged: true
+      serviceAccountName: nginx-ingress-service-account 
       containers:
         - name: nginx-ingress-controller
-          image: quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.20.0
+          image: quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.10.2
```

Comment:
Let's use `quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.21.0` . I updated it under the actual config file but never updated the README.

Reply:
Got it - thank you!